### DISCUSSION TITLE
Testando aplicativos Flutter em dispositivos iOS usando o Windows

### DISCUSSION THREAD

**Discussion by daniloasdotcom** [Flutterando/forum#201] [Perguntas e Respostas]
Sou novo no desenvolvimento de aplicativos móveis com o Flutter e atualmente estou criando códigos para um aplicativo móvel usando uma máquina Windows onde edito os códigos usando o VSCode. Neste momento eu testo meus aplicativos em um dispositivo Android (o meu telefone). No entanto, tenho algumas perguntas sobre o desenvolvimento em dispositivos iOS:

Meu código atual, que roda em dispositivos Android, pode ser executado, diretamente e sem modificações, em um dispositivo móvel iOS (iPhone, para ser mais específico)?
Preciso ter um iPhone para testar meu aplicativo no iOS?
Se eu tiver um IPhone posso então instalar meu aplicativo nele usando meu laptop Windows?

Qualquer orientação ou conselho seria muito apreciado. Obrigado!

Obs: Tenho cogitado comprar um iPhone para usar como dispositivo físico para realizar a instalação do meu aplicativo e seguir realizando testes usando um IPhone próprio. No entanto, não tenho certeza se conseguirei instalar o aplicativo de forma eficaz no IPhone conectando o iPhone ao meu laptop Windows.

**Comment by TaylorGarcia554** [comment]
Fala man, eu tbm estava com a mesma dúvida mas só adiantando. Não  consegue buildar app pra iOS no windows ou linux. So em MacOS mesmo.. Ou alugar um e rodar ne´

**Reply by daniloasdotcom** [reply]
Fala, Taylor!
Obrigado pela orientação. Estou agora cogitando adquirir um Mac Mini para suprir tanto a demanda de um MacOS para o build e publicação na Apple Store quanto para emular dispositivos móveis iOS para realizar os testes de uso.
